src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -198,3 +198,69 @@ void mongoListSubscriptions
   *oe = OrionError(SccOk);
   return;
 }
+
+
+/* ****************************************************************************
+*
+* mongoGetSubscription -
+*/
+void mongoGetSubscription
+(
+  ngsiv2::Subscription                *sub,
+  OrionError                          *oe,
+  const std::string&                  idSub,
+  std::map<std::string, std::string>& uriParam,
+  const std::string&                  tenant
+)
+{
+
+  bool           reqSemTaken = false;
```

Comment:
SGP: many spaces, two spaces (between type and var-name) is what the style guide recommends